skore/src/skore/persistence/item/pickle_item.py

```diff
@@ -70,7 +71,26 @@ def factory(cls, object: Any, /, **kwargs) -> PickleItem:
 
     def as_serializable_dict(self):
         """Get a JSON serializable representation of the item."""
+        try:
+            object = self.object
+        except Exception as e:
+            value = "UnpicklingError"
+            traceback = "".join(format_exception(None, e, e.__traceback__))
+            note = "".join(
+                (
+                    (self.note or ""),
+                    "\n\n",
+                    "UnpicklingError with complete traceback:",
+                    "\n\n",
+                    traceback,
+                )
+            )
```

Comment:
I don't love exposing such technical details in the UI itself, especially by overwriting user-made data like notes (they will be saved in the DB and take up space for nothing)

I don't really have a better solution though.